pose_frame_semantics/tutorial.md

```diff
@@ -44,9 +105,31 @@ A new `/@relative_to` attribute for `//pose` tags to specify the relative relati
 </sdf>
 ```
 
-### `//frame/@attach_to`
+For more details, see the [Pose and Frame Semantics](#pose-and-frame-semantics)
+section of this document.
+
+### Named frames
+
+Three things are needed to create a frame using the `//frame` element:
 
-A new `/@attach_to` attribute for `//frame` tags to specify the parent frame of the current frame.
+* in `//frame/@name`: a unique name.
+* in `//frame/@attached_to`: the name of a parent frame to which this frame is
+  attached. If `@attached_to` is not specified, the frame will be attached to
+  the model frame (see next section).
+* in `//frame/pose`: a pose, which may be defined relative to another frame.
+  Other links and joints are considered frames so they can be used here.
```

Comment:
Do you mean that other joints and links can also have a `//frame/pose` attribute?

Reply:
I meant that the name of a link or joint could be used in `//frame/pose/@relative_to`. This could be clarified.

Reply:
hopefully https://github.com/gazebosim/sdf_tutorials/pull/83/commits/830ec2d49091c90f2356ba15de4a29d9b690e225 is an improvement